examples/rbac.yaml

```diff
@@ -37,6 +37,18 @@ roleRef:
   kind: Role
   name: tekton-triggers-example-minimal
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
Instead of adding a binding to the admin role, could we add the [rule](https://github.com/tektoncd/triggers/blob/master/config/200-role.yaml#L30-L33) of using pod security policies to the role defined above in this file?